keyboards/minidox/keymaps/norman/keymap.c

```diff
@@ -0,0 +1,69 @@
+#include QMK_KEYBOARD_H
+
+#include <sendstring_norman.h>
+
+
+extern keymap_config_t keymap_config;
+
+// Each layer gets a name for readability, which is then used in the keymap matrix below.
+// The underscores don't mean anything - you can have a layer called STUFF or any other name.
+// Layer names don't all need to be of the same length, obviously, and you can also skip them
+// entirely and just use numbers.
+enum layers {
+    _NORMAN,
+    _LOWER,
+    _RAISE,
+    _ADJUST
+};
+
+enum custom_keycodes {
```

Comment:
Your keymap isn't using this code block at all; Lines 19-24 can be safely deleted.